src/content/ui-api/management-api/authentication/automation/index.md

```diff
@@ -0,0 +1,162 @@
+---
+linkTitle: In automation
+title: Authentication for the Management API in automation
+description: Using the Management API from an automation requires a hands-free way to provide credentials to kubectl or any Kubernetes client. This article explains how to obtain a service account token to use in such a scenario.
+weight: 20
+menu:
+  main:
+    identifier: uiapi-managementapi-authentication-automation
+    parent: uiapi-managementapi-authentication
+last_review_date: 2022-04-28
+user_questions:
+  - How can I use the Management API in a programmatic way?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rainbow
+---
+
+Using the Management API from an automation requires a hands-free way to provide credentials to kubectl or any Kubernetes client. This article explains how to make use of a service account and create a self-contained kubeconfig file to use in such a scenario.
+
+To learn about authentication as a user who can complete an interactive authentication flow in the browser, please head to our [according article]({{< relref "/ui-api/management-api/authentication/user" >}}).
+
+**Note:** In the context of this article we are assuming that you want to connect to the Management API from outside the management cluster. In case your automation is running in the management cluster, using a service for authentication becomes much simpler. TODO: explain how or point to upstream docs.
```

Comment:
```suggestion
**Note:** In the context of this article we are assuming that you want to connect to the Management API from outside the management cluster. In case your automation is running in the management cluster, using a service for authentication becomes much simpler.
```